services/web/server/src/simcore_service_webserver/scicrunch/submodule_setup.py

```diff
@@ -0,0 +1,26 @@
+"""
+    Notice that this is used as a submodule of groups'a app module
+"""
+import logging
+from typing import Any, MutableMapping, Optional
+
+from pydantic import ValidationError
+
+from ._config import SciCrunchSettings
+from .service_client import SciCrunchAPI
+
+logger = logging.getLogger(__name__)
+
+
+def setup_scicrunch_submodule(
+    app: MutableMapping[str, Any], *, cfg: Optional[SciCrunchSettings] = None
+):
+    try:
+        cfg = SciCrunchSettings()
+        api = SciCrunchAPI.acquire_instance(app, cfg)
+        assert api == SciCrunchAPI.get_instance(app)  # nosec
```

Comment:
I would not use assert and replace them with if raise

Reply:
same as before ... it is to validate an assumption that the code already should impose. In this case, the line before guarantees this condition